knexfile.js

```diff
@@ -0,0 +1,57 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: {
+      host: 'localhost',
+      user: 'alexithemia',
+      password: 'password',
+      database: 'art_and_prod'
+    }
+  },
+
+  knex: {
```

Comment:
I don't know what this is for? But it looks like you were trying to experiment with the configs. If there's no reason so keep this section of the config, it's best to delete it.